RetailCoder.VBE/UI/CodeExplorer/Commands/AddStdModuleCommand.cs

```diff
@@ -33,20 +33,20 @@ protected override bool CanExecuteImpl(object parameter)
 
         protected override void ExecuteImpl(object parameter)
         {
+            var folderAnnotation = $"'@Folder(\"{GetFolder(parameter)}\")";
```

Comment:
Hmm.. Should we have a "default folder" setting? That way it wouldn't need to be duplicated in every module-creating command...